### PR TITLE
remove duplicates and fix graph height

### PR DESCRIPTION
**Pull request by cup-noodlehS**
Issue

https://github.com/maxellmilay/finite-automata-visualizer/issues/8

### CODE REVIEW

File: components/DFA.tsx [hunk 1]
```diff
@@ -82,7 +82,7 @@ const GraphSummary = (props: PropsInterface) => {
 
     return (
         <div className="flex flex-col w-full h-full">
-            <div className="w-full h-[1000px]">
+            <div className="w-full h-full">
```
Comment (maxellmilay) [inline components/DFA.tsx:85]: @cup-noodlehS I tried this out locally but it does not display anything when height is set to a percent value

Comment (cup-noodlehS) [reply components/DFA.tsx:85]: nigga dafuk??

Comment (maxellmilay) [reply components/DFA.tsx:85]: wait let me pull this branch and double check

Comment (maxellmilay) [reply components/DFA.tsx:85]: I checked and it works, my bad

**Review by maxellmilay — APPROVED** [review]
@cup-noodlehS LGTM :fire:

**Pull request merged**